### PR TITLE
Remove "Hover" action when GameLab debug buttons are disabled

### PR DESCRIPTION
**Pull request by levadadenys**
Remove "Hover" action when GameLab debug buttons are disabled

## Links

- jira ticket: [SL-342](https://codedotorg.atlassian.net/browse/SL-342?atlOrigin=eyJpIjoiZjQ5YTFlMjYyZmQ3NGM1ZWJiZDVmZTk0ZjNkYTE4YjMiLCJwIjoiaiJ9)

upd(add imgs):

**Before:active:** 
Screenshot from 2023-01-04 20-35-15

**After:acrive:**
Screenshot from 2023-01-04 20-34-44

**Before:hover:**
Screenshot from 2023-01-04 20-49-29

**After:hover:** 
Screenshot from 2023-01-04 20-49-16

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: apps/style/JsDebuggerUi.scss [hunk 1]
```diff
@@ -29,6 +29,13 @@
   padding: 3px 6px;
   font-size: 14px;
   min-width: 120px;
+
+  &:disabled {
+    cursor: not-allowed;
+    &:active {
```
Comment (molly-moen) [inline apps/style/JsDebuggerUi.scss:35]: why was this change to `&:active` needed? Can we do it without using `!important`?

Comment (levadadenys) [reply apps/style/JsDebuggerUi.scss:35]: The problem is that we have !importnat styles in common.scss (apps/style/common.scss , line 628) that makes disabled active button to have a black border. We can leave this border for active status if it's ok and just delete this !important changes from the pr.

1. before updates 
Screenshot from 2023-01-04 20-35-15
2. after updates 
Screenshot from 2023-01-04 20-34-44

There are two more imgs I've just added to PR description for hover state of the buttons

Comment (sanchitmalhotra126) [reply apps/style/JsDebuggerUi.scss:35]: I'll defer to @moneppo for if the black border is okay - if we want to remove it, can you add a comment explaining why the !important is needed?

Comment (levadadenys) [reply apps/style/JsDebuggerUi.scss:35]: to rewrite the !important we have at (apps/style/common.scss , line 628) that makes disabled active button to have a black border.

File: apps/style/JsDebuggerUi.scss [hunk 2]
```diff
@@ -29,6 +29,15 @@
...
+      // We need this !important in order to overwrite !importnat styles in common.scss
```
Comment (molly-moen) [inline apps/style/JsDebuggerUi.scss]: nit: typo in second important

**Review by molly-moen — APPROVED** [review]
LGTM, just one typo

File: apps/style/JsDebuggerUi.scss (hunk 2 above)
Comment (levadadenys) [reply apps/style/JsDebuggerUi.scss]: Thanks!

**Review by sanchitmalhotra126 — APPROVED** [review]

**Pull request merged**